fit/core-reference/src/test/java/org/apache/syncope/fit/core/AuditITCase.java

```diff
@@ -46,35 +51,57 @@ public static UserTO getSampleUserTO(final String email) {
         return userTO;
     }
 
-    @BeforeEach
-    public void setup() {
-        /**
-         * Execute operations to activate audits.
-         */
+    private static Optional<UserTO> findUser(final String username) {
         PagedResult<UserTO> matching = userService.search(
```

Comment:
Why not simply `userService.read(username)` to replace the whole `findUser()` method?

Reply:
Done.